controllers/C_Prescription.class.php

```diff
@@ -60,17 +60,20 @@ function __construct($template_mod = "general")
             // $res = sqlStatement("SELECT * FROM drugs ORDER BY selector");
 
             $res = sqlStatement("SELECT d.name, d.ndc_number, d.form, d.size, " .
-                "d.unit, d.route, d.substitute, t.drug_id, t.selector, t.dosage, " .
-                "t.period, t.quantity, t.refills, d.drug_code " .
-                "FROM drug_templates AS t, drugs AS d WHERE " .
-                "d.drug_id = t.drug_id ORDER BY t.selector");
+            "d.unit, d.route, d.substitute, t.drug_id, t.selector, t.dosage, " .
+            "t.period, t.quantity, t.refills, d.drug_code, i.on_hand,i.expiration " .
+            "FROM drug_templates AS t " .
+            "JOIN drugs AS d ON d.drug_id = t.drug_id " .
+            "JOIN drug_inventory AS i ON d.drug_id = i.drug_id " .
+            "ORDER BY t.selector");
 
             while ($row = sqlFetchArray($res)) {
                 $tmp_output = $row['selector'];
                 if ($row['ndc_number']) {
-                    $tmp_output .= ' [' . $row['ndc_number'] . ']';
+                    $tmp_output .= ' - Price of One Tab: ' . $row['ndc_number'];
```

Comment:
hi @xpindia22 , doesn't seem right that `$row['ndc_number']` holds the price of the drug. 

https://en.wikipedia.org/wiki/National_drug_code

Reply:
Hi @stephenwaite ,
ok then in that case we can leave ndc_number for what it is supposed to display and not enter the price of one tablet during entering a drug into pharmacy record.
I will remove this "- Price of One Tab:" if you say so.